src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+class RightTriangle implements Figure {
```

Comment:
The class name 'RightTriangle' is a noun, which is good, but ensure that the class is not used to define behavior but rather to represent a right triangle entity.